packages/internal/bridge/sdk/src/tokenBridge.ts

```
@@ -551,6 +618,7 @@ export class TokenBridge {
     }
     // Withdraw request
```

should we check if it is a valid withdraw?

---

Yep good point. It could be neither a valid deposit or a valid withdrawal. In which case we'd want to throw an error I imagine